File: AirportSpaApp/ClientApp/src/models/Passenger.ts
import { Ticket } from "./Ticket";

export interface Passenger {
    id: number;
    surname: string;
    name: string;
    birthDate: string,
    patronymic?: string;
    address: string;
    series: string;
    number: string;
    ticket: Ticket[];
}

export interface PassengerFilterModel {
    flightNumber: string;
}

File: AirportApp.Domain/Entities/Passenger.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using AirportApp.Domain.Entities.Common;

namespace AirportApp.Domain.Entities
{
    public class Passenger : EntityBase
    {
        [Required]
        public string Surname { get; set; }

        [Required]
        public string Name { get; set; }

        public string Patronymic { get; set; }

        [Required]
        public DateTime BirthDate { get; set; }

        public string Address { get; set; }

        [Required]
        public string Series { get; set; }

        [Required]
        public string Number { get; set; }

        public IList<Ticket> Ticket { get; set; } = new List<Ticket>();
    }
}
